pages/design/definition-of-done.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Definition of "Done"
+---
+
+## What is this? 
+
+It is important that we in the Auto-WCAG Community Group have a shared understanding of what it means for work to be complete.  The Definition of "Done" can be used to assess when work is complete. ([The Scrum Guide](https://www.scrumguides.org/scrum-guide.html#artifact-transparency-done).
+
+It can be used by rule authors as well as reviewers as a checklist to assess whether a rule is done or if it needs another iteration.
+
+Sticking to the Definition of "Done" ensures a high quality in rules published by the Auto-WCAG Community Group, and ensures transparency in our work. 
+
+The Definition of "Done" is a living document, and might change as the rule writing community develops and matures.
+
+## The Definition of "Done"
+- The relation between the rule and its accessibility requirements is as described in the [ACT Rules Format](https://www.w3.org/TR/act-rules-format/) under [Accessibility Requirements](https://www.w3.org/TR/act-rules-format/#structure-accessibility-requirements). 
```

Comment:
Add an empty line between the heading and the list. Also, I suggest adding a line about following the style guide.